Dockerfile

```diff
@@ -1,9 +1,4 @@
-FROM alpine:latest
-
-RUN apk add --no-cache python3
-
-# Potentially move this to multi-stage build for a smaller container
-RUN python -m ensurepip --upgrade
+FROM python:3-slim
```

Comment:
I checked the version currently installed, `Python 3.10.10`

Set the image to pull from 3.10, and change the docs referencing dev instructions to reflect using /bin/bash instead of /bin/sh (Just making an assumption there of debian-slim)

Reply:
/bin/sh still works fine in debian-slim, but I did update the docs either way.